### DISCUSSION TITLE
导入react后报错react: Invalid hook call. Hooks can only be called inside of the body of a function component

### DISCUSSION THREAD

**Discussion by 0polar-aurora0** [umijs/umi#6893] [Q&A]
导入react后报错react: Invalid hook call. Hooks can only be called inside of the body of a function component

复现步骤

1. 在配置中开启mfsu

![8(Y_25}XP{Z$W3VHXJ5)ZZD](https://user-images.githubusercontent.com/38682172/124548855-e2e7c980-de60-11eb-9e1d-bdade7f70f61.png)

2.页面中导入react

![1 N3T)6XGZ)U HQ})SDDUDB](https://user-images.githubusercontent.com/38682172/124548658-97352000-de60-11eb-8c88-9924f10405a8.png)

3.报错

![VZE `S8Z`4` 7ED8B@W2B2E](https://user-images.githubusercontent.com/38682172/124548558-7bca1500-de60-11eb-92b9-21b8f5abd6be.png)

这是当前umi react本版本

![TG3F (4Q~6}@6QWWNLCFO8H](https://user-images.githubusercontent.com/38682172/124548710-ab791d00-de60-11eb-9943-6979fbccf456.png)

umi官网文档提到了这是react多实例的问题，请问没有什么具体的解决方法吗

**Comment by indown** [comment]
same issue

**Comment by ratup-top** [comment]
也是遇到相同问题，重新下载下载包后，使用useState就会报此错误

**Comment by fantasy525** [comment]
这个问题如何解决呢？
